config/initializers/gov_uk_notify.rb

```diff
@@ -0,0 +1,9 @@
+if ENV['GOVUK_NOTIFY_API_KEY'].blank?
```

Comment:
we seem to be violating DRY with the NOTIFY_KEY (twice) and the TEMPLATE_ID (3 times). Do we need this initialiser at all, as we are checking in the code anyway?